aes.go

```diff
@@ -0,0 +1,77 @@
+package wolfSSL
```

Comment:
We should add our standard wolfSSL GPLv2 copyright as a comment to the top of all source files (including examples).  Applies to all source files in this PR.

Reply:
Done